rotkehlchen/chain/ethereum/interfaces/balances.py

```diff
@@ -102,11 +106,11 @@ def addresses_with_activity(
 
         addresses_with_activity = defaultdict(list)
         for event in events:  # TODO: HistoryEventFilterquery should allow querying by type/subtype combos to avoid this  # noqa: E501
```

Comment:
Shouldn't you look at doing this TODO also though? Or at least make an issue/note about it? Is it hard?